simple-blocks/main.ts

```diff
@@ -0,0 +1,47 @@
+namespace simplified {
+
+    /**
+     * Allows your sprite to jump only if it's currently
+     * on a floor or without velocity.
+     * Expand the + to add jumping power.
+     */
+    //% group=Sprites
+    //% color="#3b6fe9"
+    //% blockId=gravity_jump
+    //% block="make $thisSprite=variables_get(mySprite) gravity jump || with power $velo"
```

Comment:
I've been trying to align all new blocks to have the shadow and variable names outside the block string
```suggestion
    //% block="make $thisSprite gravity jump || with power $velo"
    //% thisSprite.shadow=variables_get
    //% thisSprite.defl=mySprite
```